runtime/compiler/codegen/J9AheadOfTimeCompile.cpp

```diff
@@ -43,6 +43,86 @@ J9::AheadOfTimeCompile::emitClassChainOffset(uint8_t* cursor, TR_OpaqueClassBloc
    return cursor + SIZEPOINTER;
    }
 
+uintptr_t
+J9::AheadOfTimeCompile::findCorrectInlinedSiteIndex(void *constantPool, uintptr_t currentInlinedSiteIndex)
+   {
+   uintptr_t constantPoolForSiteIndex = 0;
+   uintptr_t inlinedSiteIndex = currentInlinedSiteIndex;
+
+   if (inlinedSiteIndex == (uintptr_t)-1)
+      {
+      constantPoolForSiteIndex = (uintptr_t)self()->comp()->getCurrentMethod()->constantPool();
```

Comment:
Small suggestion: save the result of self()->comp() into a local and use that everywhere.